### DISCUSSION TITLE
Summary of missing features in Terminal

### DISCUSSION THREAD

**Discussion by mgabell** [microsoft/terminal#16771] [Ideas]
Hi,

These have probably already been mentioned but things you should implement:

1. Right click menu in the terminal to be able to split the terminal into pcs.
2. Be able to save the color scheme and tab layout for next session. Its annoying to recreate the layout every time.
3. Be able to choose color and name in the same popup. Now you must go in two times in the menu.
4. Be able to resize the panes. It is horrible sometimes to see what you write.
5. Be able to select which type of split you want to make, vertical or horizontal. (See linux terminator as example)

More to come...

Br

**Comment by zadjii-msft** [comment]
I'm not entirely sure I understand 3, but the rest already have been implemented. 

1. [Right click context menu](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/profile-advanced#right-click-context-menu-preview)
2. [Behavior when starting a new terminal session](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/startup#behavior-when-starting-a-new-terminal-session)
3. Is this like, for the tab context menu?
4. [Resize a pane](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions#resize-a-pane) (though admittedly, #992 is something I plan on working on this month)
5. [`splitPane`](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions#split-a-pane) accepts a `profile` argument to choose which profile to open, or `split` to pick the direction. These actions are all under the command palette, under "Split Pane..."

**Reply by mgabell** [reply]
Yes! When you click on the Tab context menu you can choose color and also name that tab.
It would be good that you could do naming and coloring in the same popup. I think you can have a look at how outlook manages
categories. You define a name to a color, either way. Bad way in outlook is that you only have one color per name but in my opinion the color and name kinda goes hand in hand. NOT that you have to have a name...

**Reply by zadjii-msft** [reply]
I mean, if this is something you're doing on a regular basis, then maybe you'd be better served by setting the `tabColor` and `tabTitle` properties on a profile?

**Comment by Machine391** [comment]
Ok no problem.
